documentation/docs/authentication.md

```diff
@@ -180,3 +186,40 @@ Use the following options to configure the KMS server for Microsoft Entra Id:
 --jwks-uri=https://login.microsoftonline.com/<TENANT_ID>/discovery/v2.0/keys
 --jwt-audience=<CLIENT_ID>
 ```
+
+## Authenticating using an API Token
+
+The server can be configured to authenticate using an API token passed in the `Authorization` header.
+
+To proceed, follow these steps:
+
+- run Cosmian KMS server without API token authentication
+- generate a symmetric key and export it from the server
+- restart the server with the `--api-token-id` option
+- configure `ckms` client with a `kms_access_token` containing the API token in base64.
+
+To generate a new API token, use the `ckms` CLI and save the symmetric key unique identifier (<SYMMETRIC_KEY_ID>):
+
+```sh
+ckms sym keys create
+```
+
+Then export the symmetric key content:
+
+```sh
+ckms sym keys export -k <SYMMETRIC_KEY_ID> api_token.json
+```
+
+From here, you can restart the server with the `--api-token-id` option.
+
+```sh
+--api_token_id <SYMMETRIC_KEY_ID>
+```
+
+Then extract the key bytes from `api_token.json` and convert it from hex to base64:
+
+```sh
+jq -r '.value[] | select(.tag == "KeyBlock") | .value[] | select(.tag == "KeyValue") | .value[] | select(.tag == "KeyMaterial") | .value[] | select(.tag == "ByteString") | .value' api_token.json | xxd -r -p | base64
+```
```

Comment:
No interest to do this step direclty from the `ckms` command with a specific option ?

Reply:
Good idea. We could add a `base64` export format in the `ckms` export commands.
What do you thing?

Reply:
Perfect !
A lot easier than this line :D

Reply:
done in cfb2b3dfe03fe68ab1399b42e8efe0ec5420cc9d